dashboard/views.py

```diff
@@ -350,12 +350,17 @@ def grade_distribution(request, course_id=0):
     if df.empty or df['current_grade'].isnull().all():
         return HttpResponse(json.dumps({}), content_type='application/json')
 
-    df['tot_students'] = df.shape[0]
+    result = {}
```

Comment:
I feel `result` can be renamed better, variable naming should say the purpose of it. `result` doesn't fullfill that

Reply:
can this is inialized like `result = dict()` instead its more clean

Reply:
I agree about the name of the `result` variable; see my comments below. I don't particularly mind either way whether you use `dict()` or `{}`. I was taught to use the latter. @pushyamig, I'd be curious to hear why you like `dict()` better.

Reply:
My IDE somehow warns if I declare like`{}` when I change it to `dict()` it is quiet. 
<img width="540" alt="Screen Shot 2019-12-09 at 5 33 37 PM" src="https://user-images.githubusercontent.com/8579775/70478983-633e2100-1aaa-11ea-80cb-7a16e8ed4eb1.png">